src/components/Book.jsx

```diff
@@ -11,17 +11,53 @@ const Book = ({ item }) => {
     dispatch(removeBook(item.item_id));
   };
   return (
-    <div>
-      <h3>{item.title}</h3>
-      <p>
-        Author:
-        {' '}
-        {item.author}
-      </p>
-      <button type="button" onClick={handleDelete}>
-        Delete
-      </button>
-    </div>
+    <section className="contBookSurrounding">
+      <div className="contBook">
+        <span className="category">Action</span>
+        <h3 className="Title">{item.title}</h3>
+        <span className="Suzanne-Collins">
+          Author:
+          {' '}
+          {item.author}
+        </span>
+        <div className="allButtons">
+          <button className="CommentsButton" type="button">Comments</button>
+          <button className="RemoveButton" type="button" onClick={handleDelete}>
+            Remove
+          </button>
+          <button className="EditButton" type="button">Edit</button>
+        </div>
```

Comment:
👍 Good job with the layout! However, there are a couple of improvements needed to match the design precisely:

- [ ] Firstly, as per the design, there should be no horizontal line after the last button, which is the 'Edit' button. Please update your code to remove the line after the 'Edit' button, aligning it with the design.

- [ ] Secondly, the horizontal lines should be evenly spaced between the buttons. Currently, they are not evenly spaced. Please adjust your code to ensure that the horizontal lines are evenly distributed between each button, adhering to the design specifications.

| Your | Design |
| --- | --- |
|  ![image](https://github.com/eamador1/bookstore-redux/assets/55840999/03227746-a584-464b-b0fb-5e9777ce0f7b)| ![image](https://github.com/eamador1/bookstore-redux/assets/55840999/d1ae7e84-7bda-4bc6-af6a-d0404077348c)|

These adjustments will help your layout closely match the design, enhancing its visual consistency. Great work, and keep refining the details! 🎨👍✨